docs/docs/developers/tutorials/codealong/contract_tutorials/counter_contract.md

```diff
@@ -7,6 +7,8 @@ import Image from "@theme/IdealImage";
 
 In this guide, we will create our first Aztec.nr smart contract. We will build a simple private counter, where you can keep your own private counter - so no one knows what ID you are at or when you increment! This contract will get you started with the basic setup and syntax of Aztec.nr, but doesn't showcase all of the awesome stuff Aztec is capable of.
 
+This tutorial is compatible with the Aztec version `#include_aztec_version`. If you are using a different version or network, you can find the relevant tutorial by clicking the version dropdown at the top of the page.
```

Comment:
```suggestion
This tutorial is compatible with the Aztec version `#include_aztec_version`. If you are using a different version or network, you can find the relevant tutorial by clicking the version dropdown at the top of the page.
```

I think it's worth adding a bit to all of these reminders that folks can install the appropriate version with `aztec-up #include_aztec_version`, since these tutorials may be some of the first content that the encounter. They may not be familiar with how to do that yet, better to over communicate i think